docs/application/dotnet/guides/app-management/app-controls.md

```diff
@@ -486,18 +486,11 @@ You can allow other applications to launch your application and use your
 application features through application controls by exporting your
 application control functionalities. To allow other applications to
 launch your application implicitly without the application ID, declare
-your application control information in the `tizen-manifest.xml` file:
+your application control information in the Visual Studio.
```

Comment:
declare your application control information in the Visual Studio.

Please rephrase to:

specify your application control information in Visual Studio.

Reply:
done